deploy/mydevil.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+
+npm8 install
```

Comment:
You have a small syntax mistake here

Reply:
You mean npm8 binary? That's mydevil specific, due to maintaining multiple versions of node

Reply:
Ok, I have not heard about it before :)